src/config.ts

```diff
@@ -0,0 +1,73 @@
+import type {MergeableRecord} from './util/objects'
+import {deepMergeObject} from './util/objects'
+import path from 'path'
+import fs from 'fs'
+import {log} from './util/log'
+import {getParentDirectory} from './util/files'
+import Joi from 'joi'
+
+export interface FlowrConfigOptions extends MergeableRecord {
+	ignoreSourceCalls: boolean
```

Comment:
we should at least document here, in a comment, what the purpose of the flag should be.

Furthermore, we may add documentation oft his to the wiki (you can defer the issue if you do not want to do this right now).